airflow/exceptions.py

```diff
@@ -110,6 +110,18 @@ def __str__(self) -> str:
         return f"unmappable return type {type(self.value).__qualname__!r}"
 
 
+class UnmappableXComLengthPushed(AirflowException):
+    """Raise when the pushed value is to large to map as a downstream's dependency."""
+
+    def __init__(self, value: Sized, max_length: int) -> None:
+        super().__init__(value)
+        self.value = value
+        self.max_length = max_length
+
+    def __str__(self) -> str:
+        return f"unmappable return value size: {len(self.value)} > {self.max_length}"
```

Comment:
Would rename size to length for clarity

```suggestion
        return f"Unmappable return value length: {len(self.value)} > {self.max_length}"
```

Reply:
Thanks! I missed changing this.